package.json

```diff
@@ -9,6 +9,7 @@
     "deasync": "^0.1.19",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jquery": "^3.5.0",
```

Comment:
As far as I can tell this dependency is never used. See https://stackoverflow.com/a/29537014. Alternatively you can remove the dependency.